client/app/scripts/superdesk/upload/image-crop-directive.js

```diff
@@ -0,0 +1,132 @@
+define([
+    'lodash',
+    'bower_components/jcrop/js/jquery.Jcrop'
+], function(_) {
+    'use strict';
+
+    return ['notify', 'gettext', '$timeout', 'imageFactory', function(notify, gettext, $timeout, imageFactory) {
+        return {
+            scope: {
+                src: '=',
+                cords: '=',
+                boxWidth: '=',
+                boxHeight: '=',
+                aspectRatio: '=',
+                minimumSize: '=',
+                cropSelect: '=',
+                showMinSizeError: '='
+            },
+            link: function(scope, elem) {
+
+                var bounds, boundx, boundy;
+                var rwidth, rheight;
+                var minimumSize, updateTimeout;
+                var cropSelect = [];
+
+                minimumSize = scope.minimumSize ? scope.minimumSize : [200, 200];
+                cropSelect = scope.cropSelect ? getCropSelect(scope.cropSelect) : [0, 0, scope.boxWidth, scope.boxHeight];
+
+                /**
+                * Push value in clockwise sequence from Left, Top, Right then Bottom (L-T-R-B).
+                */
+                function getCropSelect(cropImage) {
+                    cropSelect.length = 0;
+
+                    cropSelect.push(cropImage.CropLeft);
+                    cropSelect.push(cropImage.CropTop);
+                    cropSelect.push(cropImage.CropRight);
+                    cropSelect.push(cropImage.CropBottom);
+
+                    return cropSelect;
+                }
+
+                // To adjust preview box as per aspect ratio
+                if (scope.aspectRatio) {
+                    if (scope.aspectRatio.toFixed(2) === '1.33') {
+                        rwidth = 300; rheight = 225;
+                    } else if (scope.aspectRatio.toFixed(2) === '1.78') {
```

Comment:
Perhaps just add two line comments, explaining that these two values represent ratios of 4/3 and 16/9, respectively.